src/wmtk/operations/VertexLaplacianSmooth.cpp

```diff
@@ -0,0 +1,29 @@
+#include "VertexLaplacianSmooth.hpp"
+
+#include <wmtk/Mesh.hpp>
+#include <wmtk/SimplicialComplex.hpp>
+
+
+namespace wmtk::operations {
+
+VertexLaplacianSmooth::VertexLaplacianSmooth(Mesh& m, const MeshAttributeHandle<double>& handle)
+    : AttributesUpdateBase(m)
+    , m_attibute_handle(handle)
+{}
+
+std::vector<Simplex> VertexLaplacianSmooth::execute(const Simplex& simplex)
+{
+    auto accessor = mesh().create_accessor<double>(m_attibute_handle);
+    const std::vector<Simplex> one_ring =
+        SimplicialComplex::vertex_one_ring(mesh(), simplex.tuple());
+    auto p_mid = accessor.vector_attribute(simplex.tuple());
+    p_mid = Eigen::Vector3d::Zero();
```

Comment:
This code needs to be fixed to work for any dimension and any type of mesh but we can do that later on.